pimsviewer/widgets.py

```diff
@@ -31,7 +32,7 @@ def fps(self):
         return 1 / self._interval
     @fps.setter
     def fps(self, value):
-        if value is None:
+        if value is None or math.isnan(value):
```

Comment:
Thank you for the suggestion and my apologies for my late reply. I have edited the code. 👍